app/controllers/general.php

```diff
@@ -543,8 +543,16 @@ function router(App $utopia, Database $dbForConsole, callable $getProjectDB, Swo
             $response->addHeader('Strict-Transport-Security', 'max-age=' . (60 * 60 * 24 * 126)); // 126 days
         }
 
+        $responseVersion = System::getEnv('_APP_VERSION', 'UNKNOWN');
```

Comment:
Since this is going to be public, we should use the [`APP_VERSION_STABLE`](https://github.com/appwrite/appwrite/blob/a30d06bc585390658f2ef6677267756852c9a44e/app/init.php#L116) constant